boskos/handlers/handlers.go

```diff
@@ -390,3 +410,35 @@ func handleMetric(r *ranch.Ranch) http.HandlerFunc {
 		res.Write(js)
 	}
 }
+
+// retryOnConflict is a copy of https://godoc.org/k8s.io/client-go/util/retry#RetryOnConflict
```

Comment:
it looks like this is really a copy of `retry.OnError` except that you curried the `retriable` parameter.

Couldn't we instead do this?
```golang
func retryOnConflict(backoff wait.Backoff, fn func() error) error {
	return retry.OnError(backoff, isConflict, fn)
}
```

Or am I missing something?

Reply:
yeah, `retry.RetryOnError` is not available in the client-go version we currently use. I am working on another PR that will also bump the kube dependencies, once that is in I could change this to use `retry.OnError`: https://github.com/kubernetes/test-infra/pull/16524

Reply:
are you working on a follow-up PR now that https://github.com/kubernetes/test-infra/pull/16524 is in?

Reply:
to answer my own question: https://github.com/kubernetes/test-infra/pull/16698